tests/integration/proxyIdentity.test.ts

```diff
@@ -0,0 +1,116 @@
+  const baseMetadataUri = 'https://token-cdn-domain/';
+  let device: Contract;
+
+  let oemClaims: IClaimsUser;
+  let installerClaims: IClaimsUser;
+  let ownerClaims: IClaimsUser;
+
+  let installerDoc: DIDDocumentFull;
+  let oemDoc: DIDDocumentFull;
+  let ownerDoc: DIDDocumentFull;
+
+  const claimData = {
+    type: 'lithium',
+    status: 'installed',
+  };
+
+  before(async () => {
+    oemDid = `did:${Methods.Erc1056}:${await oem.getAddress()}`;
+    installerDid = `did:${Methods.Erc1056}:${await installer.getAddress()}`;
+    ownerDid = `did:${Methods.Erc1056}:${await owner.getAddress()}`;
+
+    const erc1056Creator = new ContractFactory(erc1056Abi, erc1056Bytecode, deployer);
+    erc1056 = await erc1056Creator.deploy();
+
+    const erc1155Creator = new ContractFactory(multiProxyAbi, multiProxyBytecode, deployer);
+    erc1155 = await erc1155Creator.deploy(baseMetadataUri);
+
+    const proxyFactoryCreator = new ContractFactory(proxyFactoryAbi, proxyFactoryBytecode, bebat);
+    proxyFactory = await proxyFactoryCreator.deploy(erc1056.address, erc1155.address);
+
+    const store = new DidStore(await spawnIpfsDaemon());
+    const resolverSettings = await getSettings([
+      await oem.getAddress(), await installer.getAddress(), await owner.getAddress(),
+    ]);
+
+    oemDoc = new DIDDocumentFull(oemDid, new Operator(oem, resolverSettings));
+    await oemDoc.create();
+
+    installerDoc = new DIDDocumentFull(installerDid, new Operator(installer, resolverSettings));
+    await installerDoc.create();
+
+    ownerDoc = new DIDDocumentFull(ownerDid, new Operator(owner, resolverSettings));
+
+    oemClaims = new ClaimsUser(oem, oemDoc, store);
+    installerClaims = new ClaimsUser(installer, installerDoc, store);
+    ownerClaims = new ClaimsUser(installer, ownerDoc, store);
+  });
+
+  it('BEBAT creates proxy identity and becomes its owner', async () => {
+    device = await createProxy(proxyFactory, uid);
+    expect(await device.owner()).equal(await bebat.getAddress());
+    expect(parseInt(await erc1155.balanceOf(await bebat.getAddress(), uid), 16)).equal(1); // owner is the one who has non-zero balance
+  });
+
+  it('BEBAT transfers ownership to OEM', async () => {
+    await erc1155.connect(bebat).safeTransferFrom(await bebat.getAddress(), await oem.getAddress(), uid, 1, '0x0');
+    expect(await device.owner()).equal(await oem.getAddress());
+    expect(parseInt(await erc1155.balanceOf(await oem.getAddress(), uid), 16)).equal(1);
+  });
+
+  it('OEM updates Battery metadata', async () => {
+    const baseUri = 'https://cache-server-domain/';
+    await erc1155.updateUri(baseUri);
+    expect(await erc1155.uri(uid)).equal(`${baseUri}${uid}.json`);
+  });
+
+  it('OEM as the owner adds Installer to approved agents', async () => {
+    await erc1155.connect(oem).setApprovalForAll(await installer.getAddress(), true);
+    expect(await erc1155.isApprovedForAll(await oem.getAddress(), await installer.getAddress())).true;
+  });
+
```

Comment:
please add a test for: 
- Installer is able transfer the ownership from OEM to AssetOwner